deepspeed/module_inject/replace_module.py

```diff
@@ -7,6 +7,7 @@
 from ..constants import INFERENCE_GENERIC_MODE, INFERENCE_SPECIALIZED_MODE
 from ..runtime.weight_quantizer import WeightQuantization
 from torch import nn
+from torch import distributed as dist
```

Comment:
We introduced a new DS comms interface in https://github.com/microsoft/DeepSpeed/commit/36ad3119d5420c6b25b456626a40f4ded8c05a81, and are moving away from direct `torch.distributed` imports unless necessary. 

Can you change this to: `from deepspeed import comm as dist`

Reply:
Sure! I noticed both in the codebase and wasn't sure which to do. I guess I chose incorrectly. I'll push a fix shortly.